code42.json

```diff
@@ -1924,6 +1924,964 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "hunt file",
+            "identifier": "hunt_file",
+            "description": "Searches Code42 for a backed-up file with a matching hash and downloads it",
+            "verbose": "",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "hash": {
+                    "description": "The md5 or sha256 hash of the file to download",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "md5",
+                        "sha256"
+                    ],
+                    "default": "",
+                    "order": 0
+                },
+                "file_name": {
+                    "description": "The name to give to the file after it is downloaded. If not supplied, the hash will be used.",
```

Comment:
But hash is a required argument?  Does that mean if I supply a file name, it ignores the hash? If that is the case, perhaps hash should not be required

Reply:
No, the file name isn't used in search at all, it only controls what the file should be named after it's download. The hash is required and is the only thing used to look up the file. What this means is that if you don't supply a name for the file, the hash will be the name given to it.

Reply:
Yes, I remember this from XSOAR now, got it